src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -730,5 +730,253 @@ unsigned int ThermalCluster::precision() const
 {
     return 0;
 }
+
+bool ThermalCluster::LoadFromProperty(ThermalCluster& cluster, const IniFile::Property* p)
```

Comment:
As **LoadFromProperty** is a method of **ThermalCluster** class, you don't need the argument **ThermalCluster& cluster**.

But thing is I'm not sure adding this method to **ThermalCluster** class is a good thing : 
**ThermalCluster** is supposed to contain only business data and logic.
Adding a method to read or print data for a cluster goes out of the scope of business logic. 
**ThermalCluster** is already quite big and contains things it shouldn't. 
As examples : methods **markAsModified()** or **memoryUsage()** should not be there.
adding a load method makes **ThermalCluster** grow in a sense we don't want